Geometry_Engine/Modify/CleanPolyline.cs

```diff
@@ -36,14 +36,19 @@ public static partial class Modify
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Returns a polyline defined by the points which result in deviations from a straight line only. For example, if a polyline has 3 points in a straight line, the middle point is removed as part of this cleaning process. This is designed for closed polylines only")]
+        [Description("Returns a polyline defined by the points which result in deviations from a straight line only. For example, if a polyline has 3 points in a straight line, the middle point is removed as part of this cleaning process if distance betwen point is less than distanceTolerance. This is designed for closed polylines only")]
         [Input("polyline", "The polyline you wish to clean by removing unnecessary points")]
-        [Input("tolerance", "The tolerance of the angle that defines a straight line. Default is set to the value defined by BH.oM.Geometry.Tolerance.Angle")]
+        [Input("angleTolerance", "The tolerance of the angle that defines a straight line. Default is set to the value defined by BH.oM.Geometry.Tolerance.Angle")]
+        [Input("distanceTolerance", "The distance between two point that should be removed. Default is set to the value 0.02")]
         [Output("polyline", "The cleaned polyline")]
         public static Polyline CleanPolyline(this Polyline polyline, double angleTolerance = Tolerance.Angle, double distanceTolerance = Tolerance.Distance)
+
+        // I recommend we replace name from distanceTolerance to minimumSegmentLength as below ,I will keep right now as we need to fix aslo XML Toolkit
```

Comment:
Remove this comment, it is not suitable for this platform.

Reply:
Comment still needs removing. It is right to capture the work that needs doing, but it should be in a GitHub issue or discussion, not a comment in code.